services/traction/api/db/models/v1/governance.py

```diff
@@ -21,14 +21,14 @@
 from sqlalchemy.dialects.postgresql import TIMESTAMP, ARRAY, UUID
 from sqlmodel.ext.asyncio.session import AsyncSession
 
-from api.db.models.base import BaseModel
+from api.db.models.base import BaseModel, StatefulModel
 
 from api.endpoints.models.v1.errors import (
     NotFoundError,
 )
 
 
-class SchemaTemplate(BaseModel, table=True):
+class SchemaTemplate(StatefulModel, table=True):
```

Comment:
here you would just inherit from both `StatefulModel` and `TimestampModel`

Reply:
not sure we'd be having stateful model without timestamps? 🤷